pkgs/desktops/gnome-3/devtools/gnome-builder/default.nix

```diff
@@ -0,0 +1,78 @@
+{stdenv
+, clang
+, devhelp
+, fetchgit
+, gettext
+, gspell
+, gnome_themes_standard
+, gtk3
+, gtksourceview
+, json_glib
+, jsonrpc_glib
+, libdazzle
+, libgit2-glib
+, libpeas
+, libxml2
+, llvmPackages
+, meson
+, ninja
+, python3
+, python3Packages
+, pcre
+, pkgconfig
+, pkgs
+, sysprof
+, template_glib
+, vala
+, vte
+, webkitgtk
+, wrapGAppsHook
+, ...}:
+
+let
+  version = "3.27.2";
+  name = "gnome-builder-${version}";
+  pythonPath = with python3Packages; python3Packages.makePythonPath [ libpeas pygobject3 ];
```

Comment:
`libpeas` is not marked as a Python module, so it won't be added to the `pythonPath`. Packages that provide bindings should be added to `python-packages.nix` and use `toPythonModule` function.

Reply:
Ok, still in the state where I just got it to (finally) got it to startup. I'll work on that next.

Reply:
`libpeas` doesn't seem to actually provide any `.py` files. It is just a shared lib that supports loading python addons.